cmd/lima

```diff
@@ -30,4 +30,5 @@ fi
 if [ -n "${LIMA_WORKDIR}" ]; then
   set - --workdir "$LIMA_WORKDIR" "$@"
 fi
+export MSYS2_ARG_CONV_EXCL="*"
```

Comment:
This is bash extension, please use the classic sh form, or just replace the shebang to `/bin/bash`
```suggestion
# (please add a comment line here to explain this variable)
MSYS2_ARG_CONV_EXCL="*"
export MSYS2_ARG_CONV_EXCL
```

Reply:
Sorry, only tried with `dash` - maybe it has extensions over traditional bourne shell

Reply:
Oops, missed the comment line. Will update again.